src/phoenix/server/api/types/Trace.py

```diff
@@ -44,13 +43,12 @@ async def spans(
             spans = await session.scalars(
                 select(models.Span)
                 .join(models.Trace)
-                .filter(models.Trace.trace_id == self.trace_id)
+                .filter(models.Trace.id == self.trace_rowid)
```

Comment:
this is my bad (or autocomplete). filter is legacy syntax
```suggestion
                .where(models.Trace.id == self.trace_rowid)
```

Reply:
addressed in ea0b8a3ff868118ad08997aebea2efa2c72dfdbb